message_tagging_service/consumer.py

```diff
@@ -86,9 +86,14 @@ def consume(msg):
             '{name}:{stream}:{version}:{context}'.format(**mbs_msg))
     else:
         try:
+            logger.info('Start to handle build: %s:%s:%s:%s',
+                        mbs_msg['name'], mbs_msg['stream'],
+                        mbs_msg['version'], mbs_msg['context'])
+
             tagging_service.handle(rule_defs, mbs_msg)
-        except:  # noqa
+        except Exception as e:
             logger.exception(f'Failed to handle message {mbs_msg}')
+            logger.error('Reason: %s', str(e))
```

Comment:
Does this actually give more information? I'd imagine the `logger.exception` call above to display the full stack trace including the exception.